packages/gmail/src/Utils.js

```diff
@@ -99,6 +99,104 @@ export async function getMessageContent(message, desiredContent) {
   }
 }
 
+// ---
+// Send message start
+// ---
+
+export async function sendMessageWithAttachments(message) {
+  const BOUNDARY = '----=_Part_0_123456789.123456789';
```

Comment:
This const should really go at the top of the file, rather than inside this function